### PR TITLE
clean grpc response

### PR DESCRIPTION
**Pull request by sakamoto-souta**
# Summary
clean grpc response
# Purpose
avoid having content in the return value that duplicates the content of the error.
# Contents
erase `message` and `is_ok` in  the return value of responses
# Testing Methods Performed
make test t=./manage_container p=medium

### CODE REVIEW

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go [hunk 1]
```diff
@@ -345,21 +296,16 @@ func (s *server) GetDataList(ctx context.Context, in *pb.GetDataListRequest) (*p
...
 	}
+
 	getDataList, err := s.m2dbclient.GetDataList()
 	if err != nil {
 		AppLogger.Error(err)
-		return &pb.GetDataListResponse{
-			IsOk:   false,
-			Result: "",
-		}, nil
+		return nil, errors.New("")
```
Comment (arukuka) [inline packages/server/manage_container/server/libc_to_manage_container/L2m.go:305]: Could we use a following code?  
From the client side, the contents of the error would not be understood.

```suggestion
		return nil, err
```

File: packages/client/libclient-py/quickmpc/qmpc_server.py [hunk 2]
```diff
@@ -138,10 +138,10 @@ def __futures_result(
             logger.error(e)
 
         for b in response:
-            if hasattr(b, "is_ok"):
-                is_ok &= b.is_ok
-            else:
+            try:
                 is_ok &= b["is_ok"]
+            except Exception:
+                pass
```
Comment (arukuka) [inline packages/client/libclient-py/quickmpc/qmpc_server.py:144]: Since `is_ok` no longer exists, we could remove the code block here.

**Review by arukuka — CHANGES_REQUESTED** [review]
Thank you for your contribution!
I have commented on some points that I am concerned, so please check it out.

File: proto/libc_to_manage_container/libc_to_manage.proto
Comment (KotaTakahashi9320) [inline proto/libc_to_manage_container/libc_to_manage.proto:1]: It seems that is_ok of GetComputationResultResponse is also unnecessary.

**Review by KotaTakahashi9320 — CHANGES_REQUESTED** [review]
Thank you for contribution.
I commented.

File: proto/libc_to_manage_container/libc_to_manage.proto
Comment (sakamoto-souta) [reply proto/libc_to_manage_container/libc_to_manage.proto:1]: It means that the following sending is unnecessary, isnt' it?
https://github.com/acompany-develop/QuickMPC/blob/3a4c8ce6b77fd967dd69f3c4297feb3d8cee3d84/packages/server/manage_container/server/libc_to_manage_container/L2m.go#L266-L272

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go [hunk 3]
```diff
@@ -66,10 +67,7 @@ func (s *server) SendShares(ctx context.Context, in *pb.SendSharesRequest) (*pb.
 
 	errToken := s.authorize(token, []string{"demo", "dep"})
 	if errToken != nil {
-		return &pb.SendSharesResponse{
-			Message: errToken.Error(),
-			IsOk:    false,
-		}, errToken
+		return &empty.Empty{}, errToken
```
[the remark this answers is not in the document]
Comment (sakamoto-souta) [reply packages/server/manage_container/server/libc_to_manage_container/L2m.go]: [ee98e4a](https://github.com/acompany-develop/QuickMPC/pull/161/commits/ee98e4a43974b3b5a768d12b8ab2a6e8c1a2f04c)

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go (hunk 1 above)
Comment (sakamoto-souta) [reply packages/server/manage_container/server/libc_to_manage_container/L2m.go]: [fdc75cf](https://github.com/acompany-develop/QuickMPC/pull/161/commits/fdc75cf15dfdb5a18c23e339f037f4bc7bd65f57)

File: packages/client/libclient-py/quickmpc/qmpc_server.py (hunk 2 above)
Comment (sakamoto-souta) [reply packages/client/libclient-py/quickmpc/qmpc_server.py]: [17e9a7e](https://github.com/acompany-develop/QuickMPC/pull/161/commits/17e9a7e4179b7b6905083c63ed742af499ef6b97)

**Review by KotaTakahashi9320 — APPROVED** [review]
LGTM

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go [hunk 4]
```diff
@@ -79,51 +77,33 @@ func (s *server) SendShares(ctx context.Context, in *pb.SendSharesRequest) (*pb.
...
 	dataIDs := in.GetDataIds()
 	token := in.GetToken()
 
 	errToken := s.authorize(token, []string{"demo", "dep"})
 	if errToken != nil {
-		return &pb.DeleteSharesResponse{
-			Message: errToken.Error(),
-			IsOk:    false,
-		}, errToken
+		return &empty.Empty{}, errToken
```
Comment (arukuka) [inline packages/server/manage_container/server/libc_to_manage_container/L2m.go:97]:
```suggestion
		return nil, errToken
```

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go [hunk 5]
```diff
@@ -79,51 +77,33 @@ func (s *server) SendShares(ctx context.Context, in *pb.SendSharesRequest) (*pb.
...
 	}
 
 	err := s.m2dbclient.DeleteShares(dataIDs)
 	if err != nil {
 		AppLogger.Error(err)
-		return &pb.DeleteSharesResponse{
-			Message: err.Error(),
-			IsOk:    false,
-		}, nil
+		return &empty.Empty{}, err
```
Comment (arukuka) [inline packages/server/manage_container/server/libc_to_manage_container/L2m.go:103]:
```suggestion
		return nil, err
```

**Review by arukuka — COMMENTED** [review]
There were some parts I hadn't checked thoroughly :bow:,  
please check the comments.

File: packages/client/libclient-py/quickmpc/qmpc_server.py (hunk 2 above)
Comment (arukuka) [reply packages/client/libclient-py/quickmpc/qmpc_server.py]: Thank you!  checked.

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go (hunk 1 above)
Comment (arukuka) [reply packages/server/manage_container/server/libc_to_manage_container/L2m.go]: Thank you!  checked.

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (arukuka) [reply packages/server/manage_container/server/libc_to_manage_container/L2m.go]: Thank you!  checked.

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go (hunk 4 above)
Comment (sakamoto-souta) [reply packages/server/manage_container/server/libc_to_manage_container/L2m.go]: [f6973c3](https://github.com/acompany-develop/QuickMPC/pull/161/commits/f6973c36a8a03b3bf01ad35171fd58d1d7e4992d)

File: packages/server/manage_container/server/libc_to_manage_container/L2m.go (hunk 5 above)
Comment (sakamoto-souta) [reply packages/server/manage_container/server/libc_to_manage_container/L2m.go]: [f6973c3](https://github.com/acompany-develop/QuickMPC/pull/161/commits/f6973c36a8a03b3bf01ad35171fd58d1d7e4992d)

**Comment by arukuka** [comment]
@sakamoto-souta  
Please rebase this branch to latest main branch:  5d0d080e65fbacb14eb8ed9a5e5d8e86d11ecba4 :bow:

**Pull request merged**